### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by LucasUTNFRD**
- Implements the conway's game of life backend logic
- Implements the frontend to play the game
- Implement exhaustive tests for game logic
- I have problems with setting the EOL of github

### CODE REVIEW

File: src/conways.rs [hunk 1]
```diff
@@ -0,0 +1,263 @@
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum CellState {
+    Dead,
+    Alive,
+}
+
+#[derive(Debug)]
+pub struct Grid {
+    grid: Vec<Vec<CellState>>,
```
Comment (juanbono) [inline src/conways.rs:14]: can you add docs to this struct and its fields?

**Review by juanbono — CHANGES_REQUESTED** [review]
add docs

File: src/conways.rs (hunk 1 above)
Comment (LucasUTNFRD) [reply src/conways.rs:14]: Done. I added docs to conway.rs file, do you thinks is also needed in the main.rs file?

**Review by tomip01 — APPROVED** [review]
LGTM

**Pull request merged**